app/ui/src/app/integration/integration_detail/integration-description.component.ts

```diff
@@ -1,10 +1,10 @@
-import { Integration, Step } from '@syndesis/ui/platform';
+import { Flow, Integration, IntegrationType, Step } from '@syndesis/ui/platform';
 import {
   Component,
   Input,
   Output,
   EventEmitter,
-  ViewEncapsulation
+  ViewEncapsulation, OnChanges
```

Comment:
I'm surprised lint isn't complaining about this, could just be we've the line length restriction but no lint rules on import formatting perhaps.